server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcAccessor.java

```diff
@@ -101,8 +95,18 @@ public boolean openForRead() throws SQLException, SQLTimeoutException {
         if (statementRead != null && !statementRead.isClosed()) {
             return true;
         }
-
         Connection connection = super.getConnection();
+        try {
+            return openForReadInner(connection);
+        } catch (Throwable e) {
+            if (statementRead == null) {
+                JdbcBasePlugin.closeConnection(connection);

```

Comment:
```suggestion
                closeConnection(connection);

```
this method is inherited, so it is not necessary to refer to JdbcBasePlugin to invoke it

Reply:
Removed redundant reference to the class